nekoyume/Assets/_Scripts/UI/Login/PortalConnect.cs

```diff
@@ -30,6 +31,7 @@ public class RequestCodeResult : RequestResult
         public class AccessTokenResult : RequestResult
         {
             public string accessToken;
+            public string refreshToken;
```

Comment:
it doesn't seem to be used anywhere in this PR yet.

Reply:
Right. I applied the protocol changes of portal, and... there is no usage.